src/chords_simplification.py

```diff
@@ -46,18 +46,22 @@ def simplify(notes, chords, key, preview_file):
     notes (list[Sound])
     chords (list[Chord])
     key (Key)
-    preview_file (os.path-like)
 
     Returns:
     (list[Sound]) : Notes after transposition
     (list[Chord]) : Chords after transposition
     (Key) : Chosen easier key
-    (os.path-like) : Melody with chords generated
-        using transposed sounds and chords
     """
     min_score = 1000000
     min_trans = 0
-    for i in range(-11, 12):
+    lowest_note = min([x.note+x.octave*12 for x in notes if x.note])
+    highest_note = min([x.note+x.octave*12 for x in notes if x.note])
+    # lowest note that can be played on a guitar is E2, which is _note=16
+    lowest_trans = max([-11, 16-lowest_note])
```

Comment:
There is 12 notes in octave, so it should be half of octave up and down, so -6 instead of -11 and +6 instead of 12 in next line

Reply:
Done